BryantUniversity/BryantUniversity/Data/DatabaseIntializer.cs

```diff
@@ -50,36 +36,35 @@ protected override void Seed(Context context)
             SemesterDetails semEvent2 = new SemesterDetails(event2, "Advising for All students 10am – 6pm");
             SemesterDetails semEvent3 = new SemesterDetails(event3, "Advising for all students 10 A.M. – 4 P.M.");
             SemesterDetails semEvent4 = new SemesterDetails(event4, "Advising for All students 10am – 3pm");
-
             SemesterDetails semEvent5 = new SemesterDetails(event5, "Residence Halls Open");
-            
             SemesterDetails semEvent6 = new SemesterDetails(event6, "Martin Luther King, Jr. Birthday observed");
             SemesterDetails semEvent7 = new SemesterDetails(event7, "Spring 2020 Classes begin");
             SemesterDetails semEvent8 = new SemesterDetails(event8, "Add/Drop (no fee)/Late Registration ($50 fee) on the Web ");
-
             SemesterDetails semEvent9 = new SemesterDetails(event9, "Lincoln’s Birthday – classes in session; offices minimally staffed");
             SemesterDetails semEvent10 = new SemesterDetails(event10, "	President’s Day – no classes; offices closed");
             SemesterDetails semEvent11 = new SemesterDetails(event11, "Applications for graduation (Registrar’s office) due from candidates who expect to complete requirements by May 2020");
             SemesterDetails semEvent12 = new SemesterDetails(event12, "	Advising begins in department offices for Fall 2020 registration (By appointment)");
 
-            calendarRepo.Insert(semEvent1);
-            calendarRepo.Insert(semEvent2);
-            calendarRepo.Insert(semEvent3);
-            calendarRepo.Insert(semEvent4);
-            calendarRepo.Insert(semEvent5);
-            calendarRepo.Insert(semEvent6);
-
-            calendarRepo.Insert(semEvent7);
-            calendarRepo.Insert(semEvent8);
-            calendarRepo.Insert(semEvent9);
-            calendarRepo.Insert(semEvent10);
-            calendarRepo.Insert(semEvent11);
-            calendarRepo.Insert(semEvent12);
-
-            roleRepo.Insert(admin);
-            roleRepo.Insert(faculty);
-            roleRepo.Insert(student);
-       
+            context.Users.Add(adminUser);
+            context.UserRoles.Add(userRole);
+
+            context.Roles.Add(admin);
+            context.Roles.Add(faculty);
+            context.Roles.Add(researcher);
+            context.Roles.Add(student);
+
+            context.CalendarEvents.Add(semEvent1);
+            context.CalendarEvents.Add(semEvent2);
+            context.CalendarEvents.Add(semEvent3);
+            context.CalendarEvents.Add(semEvent4);
+            context.CalendarEvents.Add(semEvent5);
+            context.CalendarEvents.Add(semEvent6);
+            context.CalendarEvents.Add(semEvent7);
+            context.CalendarEvents.Add(semEvent8);
+            context.CalendarEvents.Add(semEvent9);
+            context.CalendarEvents.Add(semEvent10);
+            context.CalendarEvents.Add(semEvent11);
+            context.CalendarEvents.Add(semEvent12);
```

Comment:
If I'm not mistaken, there should be a `context.SaveChanges()` call after all this.